src/index.html

```diff
@@ -32,10 +32,11 @@
 
     <script src="../node_modules/zone.js/dist/zone.js"></script>
     <script src="../node_modules/systemjs/dist/system.js"></script>
-    <script>var require = { paths: { 'vs': '../node_modules/monaco-editor/min/vs' } };</script>
-    <script src="../node_modules/monaco-editor/min/vs/loader.js"></script>
-    <script src="../node_modules/monaco-editor/min/vs/editor/editor.main.nls.js"></script>
-    <script src="../node_modules/monaco-editor/min/vs/editor/editor.main.js"></script>
+
+    <script>var require = { paths: { 'vs': 'https://unpkg.com/monaco-editor@0.10.0/min/vs' } };</script>
```

Comment:
Why are you changing this to use a cdn? Let's not use a cdn unless we absolutely have to, especially if we don't control it or understand how it is updated.

Reply:
@MIchaelMainer While I share your concern about taking a hard dependency on CDN, instead of using it purely as a perf optimization, we need to get an update out for Graph Explorer.  So we either need to rollback the Monaco changes, or hope this fixes it.  

@jobala We should try and understand why we can't load monaco from a local module.   It seems like something we should be able to do.